integration-tests/scaffolding/helpers.ts

```diff
@@ -262,15 +270,89 @@ export async function setEpochLength(keys: KeyringPair, epochLength: number): Pr
   }
 }
 
-export async function createGraphChangeSchema(): Promise<u16> {
-  const [createSchemaEvent, eventMap] = await ExtrinsicHelper
-    .createSchema(devAccounts[0].keys, AVRO_GRAPH_CHANGE, "AvroBinary", "OnChain")
-    .fundAndSend();
-  assert.notEqual(eventMap["system.ExtrinsicSuccess"], undefined);
-  if (createSchemaEvent && ExtrinsicHelper.api.events.schemas.SchemaCreated.is(createSchemaEvent)) {
-    return createSchemaEvent.data.schemaId;
+export async function getOrCreateGraphChangeSchema(): Promise<u16> {
+  if (process.env.CHAIN_ENVIRONMENT === CHAIN_ENVIRONMENT.ROCOCO_TESTNET) {
+    const ROCOCO_GRAPH_CHANGE_SCHEMA_ID: u16 = new u16(ExtrinsicHelper.api.registry, 53);
+    return ROCOCO_GRAPH_CHANGE_SCHEMA_ID;
+  } else {
+    const [createSchemaEvent, eventMap] = await ExtrinsicHelper
+      .createSchema(devAccounts[0].keys, AVRO_GRAPH_CHANGE, "AvroBinary", "OnChain")
+      .fundAndSend();
+    assert.notEqual(eventMap["system.ExtrinsicSuccess"], undefined);
```

Comment:
Non-blocking, because it's not in scope, but if you change this file...I see a lot of this line and it's always confusing to read. I'd love to see a helper function that is "assertExtrinsicSuccess(eventMap)" or something similar.